tests/labhub_test.py

```diff
@@ -155,21 +160,24 @@ def test_create_issue_cmd(self):
         labhub.REPOS = {'repository': self.mock_repo,
                         'repository.github.io': self.mock_repo}
 
-        testbot_public.assertCommand('!new issue repository this is the title\nbo\ndy',
-                              'Here you go')
+        testbot_public.assertCommand(
+            '!new issue repository this is the title\nbo\ndy',
+            'Here you go')
 
         labhub.REPOS['repository'].create_issue.assert_called_once_with(
             'this is the title', 'bo\ndy\nOpened by @None at [text]()'
         )
 
-        testbot_public.assertCommand('!new issue repository.github.io another title\nand body',
-                              'Here you go')
+        testbot_public.assertCommand(
+            '!new issue repository.github.io another title\nand body',
```

Comment:
like the others, dont have `\n` in the middle of the line.  It makes it harder for people to read the test and understand its intention.

Also these multiple line commands should use `textwrap.dedent('''...''')` or `"""..."""` so that you dont need to put `\n` in the message, and so that the reader doesnt need to check whether the strings on multiple lines are implicit string concats or have a comma, etc.  multiline strings should use Python's multiline strings syntax.

(would be a possible enhancement to errbot to add a `assertMultilineCommand` which does the `textwrap.dedent` for the test writer.)

Reply:
or `assertCommand(..., dedent=True)`

Reply:
@jayvdb Did you meant to use `repr(multiline_string)` instead? As `textwrap.dedent` will remove only the common leading whitespace from every line in text.

Reply:
```
        testbot_public.assertCommand(
            repr('''\
        !new issue repository this is the title
        first line of body
        second line of body'
        '''),
        'Here you go')
```
Somethig like this looks ugly

Reply:
omg .. of course that is ugly.  The purpose of `dedent` is to .. dedent ... it allows the string to be indented so that it follows python indentation style.

fwiw, `repr` would be completely wrong, replacing the desired newline with literal `\` and `n`.

```py
testbot_public.assertCommand(
    textwrap.dedent('''\
        !new issue repository this is the title
        first line of body
        second line of body'
    '''),
    'Here you go')
```

The use of the `\` can be removed by creating wrapper as requested, such as `assertMultilineCommand`, because then the wrapper can do the following to remove the leading blank line

```py
'\n'.join(textwrap.dedent(command).splitlines()[1:])
```

And that can be improved to only remove the first line if it is blank.

Reply:
I opened a PR https://github.com/errbotio/errbot/pull/1238. As well as made the changes here for now.